vuepress/docs/next/docs/getting-started/ent-diag.md

```diff
@@ -9,12 +9,20 @@ Diagnostic information is critical to analysis and troubleshooting, and the Enta
 | Command | Description
 | :- | :-
 | `ent app-info` | Display basic information about Kubernetes and Entando resources
-| `ent pod-info` | Display `kubectl describe` and `kubectl logs` for each Entando pod in a namespace
+|`ent bundle info`|Show status information for the bundle project|
+|`ent --debug bundle "command"` |Enable debug mode|
 | `ent diag` | Run diagnostics on the pods currently in an Entando namespace and prepare a diagnostic tar.gz
+| `ent pod-info` | Display `kubectl describe` and `kubectl logs` for each Entando pod in a namespace
 | `ent pod grep --all "error\|fail"` | Locate error messages within EntandoApp pods 
 
-The command `ent diag` executes operations to analyze, organize and output detailed pod information. In addition to calling `ent pod-info`, `ent diag` exports custom resources, ingresses, deployments, "previous" pod logs, namespace events, etc. It also generates diagnostics and returns log paths similar to the following:
+
+### Command Details
+* The command `ent diag` executes operations to analyze, organize and output detailed pod information. In addition to calling `ent pod-info`, `ent diag` exports custom resources, ingresses, deployments, "previous" pod logs, namespace events, etc. It also generates diagnostics and returns log paths similar to the following:
 ```
 > Collected diagdata available under "~/.entando/reports/entando-diagdata-2020-11-19T02:58:47+0000" for consultation
 > Collected diagdata available in archive "~/.entando/reports/entando-diagdata-2020-11-19T02:58:47+0000.tgz"
-```
\ No newline at end of file
+```
+
+### Log Files
+* Log files for the bundle build and pack commands are generated for each component inside the .entando/logs directory. For a single component, it is printed to the standard output.  Run command logs are also printed to the standard output.
```

Comment:
played with it a bit, but there is only one log file for a single component.

I think the word 'also' means they are separate issues. there is nothing to indicate that single component and run commands are related other than where the log files. I don't think there will be confusion because it is a typical process for it to result in the stdout.